esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -651,9 +682,14 @@ def __init__(self, config):
 
         # Get default settings
         self.cfg = deepcopy(self.cfg)
+        self.cfg.setdefault('add_ancillary_variables', False)
+        self.cfg.setdefault('add_aux_coords', False)
+        self.cfg.setdefault('add_cell_measures', False)
         self.cfg.setdefault('facet_used_for_labels', 'dataset')
+        self.cfg.setdefault('facets_as_columns', [])
         self.cfg.setdefault('figure_kwargs', {'constrained_layout': True})
         self.cfg.setdefault('group_variables_by', 'short_name')
+        self.cfg.setdefault('groupby_facet', 'short_name')
```

Comment:
It looks like these newly added options are not used. I guess they have been accidentally copied over from the seaborn diagnostic?

Reply:
Not sure where those came from. Deleted the 4 unused entries.